lib/readline.js

```diff
@@ -20,26 +21,28 @@ const isFullWidthCodePoint = internalReadline.isFullWidthCodePoint;
 const stripVTControlCharacters = internalReadline.stripVTControlCharacters;
 
 
-exports.createInterface = function(input, output, completer, terminal) {
+exports.createInterface = function(input, output, completer, terminal,
+                                   crlfDelay = kcrlfDelayInMillis) {
```

Comment:
Hmm.. rather than adding an individual property here, I think I'd much rather see an options object with a crlfDelay property... makes things more scalable in the off chance we have a need to add additional options later, and it's more consistent with other core APIs